### PR TITLE
docs: add videos to getting started guides

### PR DESCRIPTION
**Pull request by mattmaribojoc**
# Related issue

Getting Started Videos were recently published, so we can add them to the relevant docs!

Closes #

# Scope of work

- adds getting started to next/nuxt getting started guides

# Screenshots of visual changes

image
image

# Checklist

- [x] Self code-reviewed
- [x] Changes documented
- [x] Semantic HTML
- [x] SSR-friendly
- [x] Caching friendly
- [x] a11y for WCAG 2.0 AA
- [x] examples created
- [x] blocks created
- [x] cypress tests created

### CODE REVIEW

File: apps/docs/components/react/getting-started.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ You can try out Storefront UI in your browser with our online playground.
 
 ## Next.js
 
+If you prefer video guides, we have a quick video that can help you set up Storefront UI in your Next.js project.
+
+<iframe src="https://www.youtube-nocookie.com/embed/6xOnCOXU8H4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="allowfullscreen" class="w-full max-w-lg mx-auto rounded aspect-video relative mt-8 custom-block"></iframe>
```
[the remark this answers is not in the document]
Comment (mattmaribojoc) [reply apps/docs/components/react/getting-started.md:24]: i thought the full width was too big, but this is just my personal preference. I also assume that most people will just full-screen the video. I don't have strong opinions, so open to changing :)

image

[the remark this answers is not in the document]
Comment (Szymon-dziewonski) [reply apps/docs/components/react/getting-started.md:24]: not sure either, for me its better full width, lets keep it your way :)

**Review by Szymon-dziewonski — APPROVED** [review]

File: apps/docs/components/react/getting-started.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (FRSgit) [reply apps/docs/components/react/getting-started.md:24]: IMO your argument @mattmaribojoc makes sense - not sure if anyone would look at this video without making it fullscreen by themselves.
Could you ask our UX/UI designer about opinion on that and act based on their input?

**Comment by Szymon-dziewonski** [comment]
@mattmaribojoc this PR can be merged and meanwhile ask designers if this is fine

**Pull request merged**